### ISSUE TITLE
Bug with a list of Charts

### ISSUE THREAD

**Issue by bobbinch** [beto-rodriguez/Live-Charts#257]
Hi,

I tried to put a list of charts up but it doesn't work. In my c# code, I have a `List<SeriesCollection> ` and here's my xaml code : 
```
<ItemsControl ItemsSource="{Binding Path=listChart}">
	<ItemsControl.ItemTemplate>
		<DataTemplate>
			<lvc:CartesianChart Width="500" Series="{Binding}">
			</lvc:CartesianChart>
		</DataTemplate>
	</ItemsControl.ItemTemplate>
</ItemsControl>
 
```
and here's the result I obtain :
![chart](https://cloud.githubusercontent.com/assets/20455652/16836689/a9ba1620-49b6-11e6-996a-7bba3e1ddfd4.PNG)

Thanks,

**Comment by beto-rodriguez** [comment]
I am really disappointed with XAML, why this!

Any ways a work around that works for me is:

```c#
    DataSource = new List<ChartValues<double>>
            {
                new ChartValues<double> {r.Next(0, 10), r.Next(0, 10), r.Next(0, 10)},
                new ChartValues<double> {r.Next(0, 10), r.Next(0, 10), r.Next(0, 10)},
                new ChartValues<double> {r.Next(0, 10), r.Next(0, 10), r.Next(0, 10)}
            };

            DataContext = this;
```

Then:

```xml
<ItemsControl ItemsSource="{Binding DataSource}">
                <ItemsControl.ItemTemplate>
                    <DataTemplate>
                        <!--Series="{Binding 
                            PresentationTraceSources.TraceLevel=High,
                            Converter={StaticResource TestConverter}}"-->
                        <lvc:CartesianChart Height="400">
                            <lvc:CartesianChart.Series>
                                <lvc:LineSeries Values="{Binding}"></lvc:LineSeries>
                            </lvc:CartesianChart.Series>
                        </lvc:CartesianChart>
                    </DataTemplate>
                </ItemsControl.ItemTemplate>
            </ItemsControl>
```

**Comment by beto-rodriguez** [comment]
Currently SeriesCollection is not a WPF UiFramework element, i guess this is causing the issue, I really didn't wanted to do this since SeriesColleciton class is in the core of the library, but since we are facing this issue, I guess I need to build a UiFrameworkSeriesCollection element

**Comment by beto-rodriguez** [comment]
This was a bug actually with every Framework element in the library, I was braking the bindings with

    SetValue()

instead of

    SetCurrentValue()

This should be fixed in 0.7.7 version, thanks for the report
